src/main/java/org/folio/circulation/domain/RequestRepresentation.java

```diff
@@ -97,6 +97,8 @@ private static void addInstanceProperties(JsonObject request, Item item) {
       write(instance, "title", item.getTitle());
       write(instance, "identifiers", item.getIdentifiers());
       write(instance, "contributorNames", mapContributorsToNamesOnly(item.getContributors()));
+      write(instance, "publication", item.getPublication());
+      write(instance, "editions", item.getEditions());
```

Comment:
Please add the new fields assertion to RequestsAPICreationTests and RequestsAPIRetrievalTests

Reply:
done